### ISSUE TITLE
Vis discount module

### ISSUE THREAD

**Issue by SmokeyTheBandicoot** [MachineMuse/MachineMusePowersuits#600]
What about a Vis discount modules that gives something like between 5% And 10% (Maybe configurable) for Thaumcraft recipes? Just like a Thaumcraft armor, it is required on all armor pieces and consumes a little energy constantly

**Comment by MachineMuse** [comment]
This would be starting to encroach on Thaumcraft's territory too much, don't you think?

Maybe if it requires infusion...

**Comment by aionys** [comment]
How about making the modules give the same discount as the TC armor, and requiring that piece of armor to install the module, kind of like with the goggles? (E.g. Leg module requires the leggings, etc.)

**Comment by SmokeyTheBandicoot** [comment]
Yes, why not, something ,like the goggles. If thaumcraft is installed, then add this module, that requires install on all components (except the power fist), that gives aech piece 5% of vis discount and requuires the respective piece of Thaumcraft's Thaumium armour. So it doesn't require any infusion, and mmmps still remains separate from Thaumcraft.

**Comment by Stalkstalks** [comment]
I just want to comment that I absolutely love magitech stuff and so I would absolutely love being able to ingrate the major magic mods with the modular armor and tool. I agree that the simplest way to do that would be to just install magic components rather than add infusion, rituals, etc, support to MMPS.

**Comment by MachineMuse** [comment]
It seems it's actually quite simple to add an infusion recipe through the API.
